tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -144,24 +144,42 @@ def test__get_collections_with_auth__OK(self):
         self.assertEqual(200, res_auth.status_code)
         self.assertEqual(6, len(res_auth.json["collections"]))
```

Comment:
These two checks above are not needed since we are testing that in the test above.